docs/api/global-shortcut.md

```diff
@@ -8,13 +8,13 @@ The `globalShortcut` module can register/unregister a global keyboard shortcut
 with the operating system so that you can customize the operations for various
 shortcuts.
 
-**Note:** The shortcut is global; it will work even if the app does
-not have the keyboard focus. This module cannot be used before the `ready`
-event of the app module is emitted.
-
-Please also note that it is also possible to use Chromium's
-`GlobalShortcutsPortal` implementation, which allows apps to bind global
-shortcuts when running within a Wayland session.
+> [!NOTE]
+> The shortcut is global; it will work even if the app does
+> not have the keyboard focus. This module cannot be used before the `ready`
+> event of the app module is emitted.
```

Comment:
```suggestion
> event of the app module is emitted.  
```

These extra spaces will keep the line break.

Reply:
I think this change in particular caused a CI failure so I had to revert it https://github.com/electron/electron/pull/46893/commits/f6eeb61d6d3682aa4e64247a8f4859992682cb11

Reply:
Okay, not a big deal to get rid of the line break. :)